go-controller/pkg/ovn/master.go

```diff
@@ -485,11 +497,51 @@ func (oc *Controller) ensureNodeLogicalNetwork(nodeName string, hostsubnet *net.
 	return nil
 }
 
+// annotate the node with the subnet information assigned to node's logical switch. the
+// new format of the annotation is:
+//
+// k8s.ovn.org/node-subnets: {
+//	  "default": "192.168.2.1",
+// }
+func (oc *Controller) addNodeAnnotations(node *kapi.Node, subnet string, replace bool) error {
+	// nothing to do if the node already has the annotation key
+	_, ok := node.Annotations[OvnHostSubnet]
```

Comment:
I was about to add a comment saying "you're checking the wrong annotation here" but then I realized that you just kept the old name for the constant. But maybe it would be clearer if you changed `OvnHostSubnet` to `OvnNodeSubnets`? Or `NodeSubnetsAnnotation` or something...

Reply:
Agree and will do. I started off with OvnNodeSubnets but the name change was at several places, so I didn't want that to distract the review of other important code changes.